src/ByteSizeLib/EnumerableExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ByteSizeLib
+{
+    public static class EnumerableExtensions
+    {
+        public static ByteSize Sum(this IEnumerable<ByteSize> byteSizes)
+        {
+            return byteSizes.Aggregate(ByteSize.FromBits(0), (current, byteSize) => current + byteSize);
```

Comment:
Why is there a seed specified?

Reply:
Because I didn't remember that the `Aggregate` method also works without a seed. 😄

Addressed in 867bb5de3c0c9c144fa872b2a308b7eab4e19025 which is freshly rebased on master.